lib/routes/account.js

```diff
@@ -772,7 +796,7 @@ module.exports = function (
       path: '/account/keys',
       config: {
         auth: {
-          strategy: 'keyFetchToken'
+          strategy: 'keyFetchTokenVerified'
```

Comment:
Is `/account/keys` called by Fx (I imagine yes, otherwise how do they get the keys)? If so, [is changing the strategy name is going to cause breakage](https://github.com/mozilla/fxa-auth-server/pull/1232/files/fc6b6c9a2789a687d00d7396d2cd544317bd1188#r60723265)?